vertical-pod-autoscaler/pkg/recommender/model/cluster.go

```diff
@@ -35,22 +35,17 @@ const (
 	RecommendationMissingMaxDuration = 30 * time.Minute
 )
 
-// ClusterState holds all runtime information about the cluster required for the
-// VPA operations, i.e. configuration of resources (pods, containers,
-// VPA objects), aggregated utilization of compute resources (CPU, memory) and
-// events (container OOMs).
-// All input to the VPA Recommender algorithm lives in this structure.
-type ClusterState struct {
+type clusterState struct {
 	// Pods in the cluster.
-	Pods map[PodID]*PodState
+	pods map[PodID]*PodState
 	// VPA objects in the cluster.
-	Vpas map[VpaID]*Vpa
+	vpas map[VpaID]*Vpa
 	// VPA objects in the cluster that have no recommendation mapped to the first
 	// time we've noticed the recommendation missing or last time we logged
 	// a warning about it.
-	EmptyVPAs map[VpaID]time.Time
+	emptyVPAs map[VpaID]time.Time
 	// Observed VPAs. Used to check if there are updates needed.
-	ObservedVpas []*vpa_types.VerticalPodAutoscaler
+	observedVpas []*vpa_types.VerticalPodAutoscaler
```

Comment:
Since you're here touching these, would it be possible to be consistent with the case of VPA?
My vote is to settle on upper-case "VPA", and not "Vpa"

Reply:
done.